api/envoy/type/matcher/v3/regex.proto

```diff
@@ -45,6 +45,12 @@ message RegexMatcher {
     //
     // This field is deprecated; regexp validation should be performed on the management server
     // instead of being done by each individual client.
+    //
+    // .. note::
+    //
+    //  Although this field is deprecated, the program size will still be checked against the
+    //  ``max_program_size`` runtime value.
```

Comment:
It's better to use a fully qualified name.
```suggestion
    //  gloabl ``re2.max_program_size.error_level`` runtime value.
```
Bear in mind there's also `re2.max_program_size.warn_level` (effectively not set by default). If the compiled regex is greater than this then a warning is issued, but the process keeps running.

Reply:
Ok